securing_apps/topics/oidc/nodejs-adapter.adoc

```diff
@@ -97,6 +128,14 @@ session store that `express-session` is using.
     const session = require('express-session');
     const memoryStore = new session.MemoryStore();
 
+    // Configure session
```

Comment:
@pacostas I believe would make sense to indent the code below. Other than that, changes lgtm